java/RSPEC-2151.java

```diff
@@ -0,0 +1,8 @@
+class Finalizers {
```

Comment:
RSPEC-2176: Class has same name as parent
<!-- lookout footnote separator -->
_Comment made by the analyzer sonarcheck_analyzer._